src/Explorer/QueryCopilot/QueryCopilotTab.tsx

```diff
@@ -240,6 +240,7 @@ export const QueryCopilotTab: React.FC<QueryCopilotTabProps> = ({
       commandButtonLabel: executeQueryBtnLabel,
       ariaLabel: executeQueryBtnLabel,
       hasPopup: false,
+      disabled: query?.trim() === "" && true,
```

Comment:
Just `query?.trim() === ""` is sufficient

Reply:
Resolved